edb/pgsql/delta.py

```diff
@@ -3425,13 +3502,7 @@ def _alter_pointer_type(self, pointer, schema, orig_schema, context):
         self._recreate_constraints(pointer, schema, context)
 
         if changing_col_type:
-            self.schedule_inhviews_update(
-                schema,
-                context,
-                source,
-                update_descendants=True,
-                update_ancestors=True,
-            )
+            self.create_inhview(schema, context, source)
```

Comment:
Is the idea that we do the batching for things like property creation that would be very expensive to do eagerly, but just do things eagerly for less hot things? (Like here?)

Reply:
Yeah.  Batching is an optimization and needs to be used sparingly and carefully, lest the original problems that we're attempting to solve here recur.

Reply:
:+1: